_data/faq.yml

```diff
@@ -219,9 +240,24 @@ you through the project set up.
 </p>"}
 
 
+    - {question: "Do I have to use open datasets for my project?", answer: "<p>
```

Comment:
@jsheunis 
For the FAQ for the hackathon.
Could you just quickly cross check that what I suggest about open data sets is not complete BS. 
Thanks